src/frontend/pages/ProjectsPage/projects-table-filter/projects-table-filter.tsx

```diff
@@ -146,7 +148,7 @@ const ProjectsTableFilter: React.FC<FilterProps> = ({ onClick, leads, managers }
 
   return (
     <>
-      <Card>
+      <Card bg={theme.cardBg} className={theme.themeName}>
```

Comment:
Do you think maybe we could have an abstracted card component the same way we have the page block component? Ideally this would significantly reduce the number of locations we're using the theme hook

Reply:
i think that would be a good idea for a future ticket. I honestly don't know how similar all their construction is though